pyincore/spaceservice.py

```diff
@@ -91,6 +109,57 @@ def update_space(self, space_id: str, space_json):
         response = r.json()
         return response
 
+    def add_to_space_by_name(self, space_name, dataset_id):
+        """Add dataset to a space by using space name and dataset id.
+
+        Args:
+            space_name (str): Name of the space that dataset id will go to.
+            dataset_id (json): ID of the dataset that will be copied in to the space.
+
+        Returns:
+            obj: HTTP response with the updated space.
+
+        """
+        space_id = self.get_space_by_name(space_name)[0]
+
+        response = self.add_dataset_to_space(space_id["id"], dataset_id)
+
+        return response
+
+    def delete_from_space_by_name(self, space_name, dataset_id):
+        """Delete dataset from a space by using space name and dataset id.
+
+        Args:
+            space_name (str): Name of the space that dataset id will go to.
+            dataset_id (json): ID of the dataset that will be copied in to the space.
+
+        Returns:
+            obj: HTTP response with the updated space.
+
+        """
+        space_id = self.get_space_by_name(space_name)[0]
+
+        response = self.delete_dataset_from_space(space_id["id"], dataset_id)
+
+        return response
+
+    def delete_dataset_from_space(self, space_id: str, dataset_id: str):
```

Comment:
Same here, remove instead of delete.

Reply:
I have changed the word delete to remove in every occurrences in the PR's change including changelog